js/interface.js

```diff
@@ -94,6 +95,43 @@ var DynamicLists = (function() {
     _this.init();
   }
 
+  function initDataSourceProvider(currentDataSourceId) {
+    var dataSourceData = {
+      dataSourceTitle: 'Your list data',
+      dataSourceId: currentDataSourceId,
+      appId: Fliplet.Env.get('appId'),
+      default: {
+        name: 'List from data source - ' + Fliplet.Env.get('appName'),
```

Comment:
https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/4743c434ff975cece766f99c77487a02fb3725e6/js/interface.js#L1627

The current implementation gives you a default data source name based on a combination of the app name and the LFD layout.

We should keep the same functionality.

Reply:
@tonytlwu  resolved, in the future I will use this structure

Reply:
@AndrRyaz Essentially: pay attention to how features work before you make the changes, and don't break them.